api/src/main/scala/uk/ac/wellcome/platform/api/controllers/MainController.scala

```diff
@@ -2,6 +2,7 @@ package uk.ac.wellcome.platform.api.controllers
 
 import com.twitter.finagle.http.Request
 import com.twitter.finatra.http.Controller
+import uk.ac.wellcome.platform.api.models._
```

Comment:
Why move this import? Is this an IntelliJ thing?

(And are there rules for import order in Scala/Java? I’ve been vaguely adapting the Python rules.)